web/src/settings.js

```diff
@@ -147,6 +148,7 @@ export function build_page() {
 
     settings_bots.update_bot_settings_tip($("#personal-bot-settings-tip"), false);
     $(".settings-box").html(rendered_settings_tab);
+    common.adjust_mac_kbd_tags("#user_enter_sends_label kbd");
```

Comment:
Should this be added in the next commit as well for realm setting?

Reply:
Right, Fixed.

Reply:
Can you point to where this is done?

Reply:
This change was originally part of a previous commit (431bc9ca1d6ced650c5b18eb6a0f63d27f9ca002) but is now included in the latest commit.

Regarding your question about adding `adjust_mac_kbd_tags` for realm settings, if that's what you meant, issue #29083 specifies the following:

> In personal settings, the setting should appear as "Return" instead of "Enter" on a Mac, similar to how it appears in the compose box menu.
> In Default user settings, "Enter" should always be used.